pkg/daemon/daemon.go

```diff
@@ -711,6 +721,11 @@ func (a GlogLogger) Logf(format string, v ...interface{}) {
 func (dn *Daemon) annotateNode(node, value string) error {
 	glog.Infof("annotateNode(): Annotate node %s with: %s", node, value)
 
+	// Check if node already contains annotation
```

Comment:
IMO this common method should not change.

caller should determine (if relevant) wether or not to set the annotation based on `dn.node` state (line 553 above)